docs/contributing/coding-guidelines/ros-nodes/directory-structure.md

```diff
@@ -11,47 +11,60 @@
 ├─ config
 │   ├─ foo_ros.param.yaml
 │   └─ foo_non_ros.yaml
+├─ doc
+│   ├─ foo_document.md
+│   └─ foo_diagram.svg
 ├─ include
 │   └─ <package_name>
-│      └─ foo_public.hpp
+│       └─ foo_public.hpp
 ├─ launch
 │   ├─ foo.launch.xml
 │   └─ foo.launch.py
+├─ schema
+│   └─ foo_node.schema.json
 ├─ src
 │   ├─ foo_node.cpp
 │   ├─ foo_node.hpp
 │   └─ foo_private.hpp
 ├─ test
 │   └─ test_foo.cpp
 ├─ package.xml
-└─ CMakeLists.txt
+├─ CMakeLists.txt
+└─ README.md
 ```
 
-### config directory
+### Config directory
```

Comment:
```suggestion
### `config` directory
```
What about this convention?

Also, instead of typing "directory" for all these folders, we can put them like:

```md
### Directory descriptions

#### `src`

#### `config`

etc.
```

Reply:
Updated sections https://github.com/autowarefoundation/autoware-documentation/pull/386/commits/b9c389ed7f8e312de694efd2ffbc9cc4af51e625.